app/src/main/java/com/example/footballmanager/screens/result_screens/success_components/MatchScoreTab.kt

```diff
@@ -0,0 +1,324 @@
+    ) {
+        val defaultValue = stringResource(id = R.string.default_value)
+        LazyColumn(
+            state = lazyListState,
+            modifier = Modifier
+                .fillMaxWidth()
+        ) {
+            var previousLeagueName = defaultValue
+            for (i in 0..lastLoadedAmount) {
+                val item = fixture.responseBody[i]
+                val leagueName = item.league?.name ?: defaultValue
+                val leagueLogo = item.league?.logo ?: defaultValue
+                val nameTeamHome = item.teams?.home?.name ?: defaultValue
+                val nameTeamAway = item.teams?.away?.name ?: defaultValue
+                val scoreTeamHome = item.goals?.home?.toString() ?: defaultValue
+                val scoreTeamAway = item.goals?.away?.toString() ?: defaultValue
+                val logoTeamHome = item.teams?.home?.logo ?: defaultValue
+                val logoTeamAway = item.teams?.away?.logo ?: defaultValue
+                val short = item.fixture?.status?.short ?: defaultValue
+
+                if (leagueName != previousLeagueName) {
+                    item {
+                        LeagueHeader(
+                            leagueLogo = leagueLogo,
+                            leagueName = leagueName
+                        )
+                    }
+                }
+                previousLeagueName = leagueName
+
+
+                item {
+                    FixtureItem(
+                        nameTeamHome = nameTeamHome,
+                        nameTeamAway = nameTeamAway,
+                        scoreTeamHome = scoreTeamHome,
+                        scoreTeamAway = scoreTeamAway,
+                        logoTeamHome = logoTeamHome,
+                        logoTeamAway = logoTeamAway,
+                        short = short,
+                        numberToAskDoNotKnowWhatItIs = "15/4"
+                    )
+
+                }
+            }
+        }
+
+    }
+}
+
+@Composable
+fun LeagueHeader(
+    leagueLogo: String,
+    leagueName: String
+) {
+    Row(
+        horizontalArrangement = Arrangement.spacedBy(
+            dimensionResource(id = R.dimen.league_header_spacing),
+            Alignment.Start
+        ),
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = Modifier
+            .fillMaxWidth()
+    ) {
+        Image(
+            painter = rememberAsyncImagePainter(leagueLogo),
+            contentDescription = stringResource(R.string.league_logo),
+            modifier = Modifier
+                .requiredSize(size = dimensionResource(id = R.dimen.league_header_image_required_size))
+                .clip(shape = RoundedCornerShape(dimensionResource(id = R.dimen.rounded_corners_radius)))
+        )
+        Text(
+            text = leagueName,
+            color = colorResource(id = R.color.league_header_text_color),
+            textAlign = TextAlign.Center,
+            lineHeight = 1.5.em, //nie wiem jak to przeniesc do res
```

Comment:
you can move it to dimens:
<dimen name="lineHeight">5em</dimen>

Reply:
to się niestety nie kompiluje